ccd-definition/CaseEvent/Camunda/DashboardEvents-R2-CUI-nonprod.json

```diff
@@ -284,5 +284,31 @@
     "ShowSummary": "N",
     "ShowEventNotes": "N",
     "RetriesTimeoutURLAboutToSubmitEvent": 0
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "CREATE_DASHBOARD_NOTIFICATION_FOR_HEARING_FEE_UNPAID_FOR_CLAIMANT",
```

Comment:
should be CLAIMANT1 as DEFENDANT1? to be consistent...

Reply:
Made the change :)